lollipop/types.py

```diff
@@ -1292,3 +1293,42 @@ def __repr__(self):
             klass=self.__class__.__name__,
             inner_type=repr(self.inner_type),
         )
+
+
+class Transform(Type):
+    """A wrapper type which allows us to convert data structures to an inner type,
+       then loaded or dumped with a customized format.
+    Example:
+
+        Point = namedtuple('Point', ['x', 'y'])
+
+        PointType = Transform(
+          Tuple(Integer(), Integer()),
+          load=lambda values: Point(values[0], values[1]),
+          dump=lambda point: [point.x, point.y],
+        )
+
+    :param Type inner_type: Data type.
+    :param load_modifier: Specifies loaded format for object.
+    :param dump_modifier: Specifies dump format for object.
+
+    Tuple([String(), Integer(), Boolean()]).load(['foo', 123, False])
+    """
+    def __init__(self, inner_type, load_modifier=None, dump_modifier=None):
+        super(Transform, self).__init__()
+        self.inner_type = inner_type
+        self.load_modifier = load_modifier
+        self.dump_modifier = dump_modifier
+
+    def post_load(self, data, *args, **kwargs):
```

Comment:
These are redundant. Just rename `load_modifier` property to `post_load` and it will work exactly the same with less code. Same applies to `dump_modifier` and `pre_dump`.